packages/core/src/util.ts

```diff
@@ -327,7 +341,9 @@ export class Util {
             await exec(command + ' --inputFile=' + SMART_INPUT_FILE);
             return await JSONStream.parse(fs.createReadStream(SMART_OUT_FILE));
             // eslint-disable-next-line no-empty
-        } catch (err) { }
+        } catch (err) { 
+            console.error('error while running smart selection mode', err)
```

Comment:
will always log when smart-binary is not present. Therefore, hadn't kept the log.

Reply:
If binary is not available, I think it won't enter the try/catch block, It should return before that https://github.com/LambdaTest/test-at-scale-js/pull/7/files#diff-bf366bef5d66802671c79bde5d11d97168d1f8c5a6888ca30792cd521c4812b8R336